include/deal.II/numerics/error_estimator.h

```diff
@@ -117,10 +117,12 @@ namespace hp
  * contributions of the faces (which are the integrated square of the jumps
  * times some factor) of each cell and take the square root.
  *
- * The integration is done using a quadrature formula on the face. For linear
- * trial functions (FEQ1), QGauss (with two points) or even the QMidpoint rule
- * will suffice. For higher order elements, it is necessary to utilize higher
- * order quadrature formulae as well.
+ * The integration is done using a quadrature formula on the face
+ * provided by the caller of the estimate() functions declared by this
+ * class. For linear trial functions (FE_Q(1)), QGauss with two points
+ * or even the QMidpoint rule might actually suffice. For higher order
+ * elements, it is necessary to utilize higher order quadrature
+ * formulae with `fe.degree+1` Gauss points.
  *
  * We store the contribution of each face in a @p map, as provided by the C++
```

Comment:
Would you mind also fixing this paragraph? We definitely do not store the errors per cell with this scheme.

Reply:
I'm actually not quite sure what you mean. It's true that the paragraph talks about things that are implementation details (which we try to avoid these days in the documentation), but the class *does* compute error indicators for each cell.

Reply:
I should have written 'face' instead of 'cell'. At first I thought this comment was for the public interface but you're right, its just the implementation.